### DISCUSSION TITLE
How to embed other website data

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#15708] [How to]
Hi,
I would like to display widget from other website with below script

<script async defer id='202221014343556' src='https://widgets.worldtimeserver.com/Public.ashx?rid=202221014343556&theme=Digital&action=clock&wtsid=IN&hex=ff9900&city=Bangalore&size=small'></script>

in file DetailViewHeaderTitle.tpl for contact module

i tried adding as below but does not work
 "<  div> 
<script async defer id='202221014343556' src='https://widgets.worldtimeserver.com/Public.ashx?rid=202221014343556&theme=Digital&action=clock&wtsid=IN&hex=ff9900&city=Bangalore&size=small'></script>
<  /div>" please ignore space in div tags

can you please assist

thanks and regards
Jeevan Pendhari

**Comment by mariuszkrzaczkowski** [comment]
the system does not allow it, several security mechanisms will block it,
this is a security breach

**Reply by JEEVAN28** [reply]
ok, thanks
